modules/index.js

```diff
@@ -0,0 +1,95 @@
+import { DateTime } from './luxon.js';
+import Book from './book.js';
+import Books from './books.js';
+import Library from './library.js';
+
+function displayBooks() {
+  const bookListDiv = document.querySelector('.book-list');
+  bookListDiv.innerHTML = '';
+  const bookList = document.createElement('ul');
+
+  Books.forEach((book, index) => {
+    const li = document.createElement('li');
+    li.setAttribute('id', book.id);
+    li.innerHTML = `
+      <p><b>"${book.title}"</b> by ${book.author}</p>
+      <button class="remove-btn">Remove</button>
+    `;
+
+    if (index % 2 === 0) {
+      li.classList.add('dark');
+    } else {
+      li.classList.add('light');
+    }
+
+    li.querySelector('.remove-btn').addEventListener('click', () => {
+      const library = new Library(book);
+      library.removeBook();
+      li.remove();
+      displayBooks();
+    });
+
+    bookList.appendChild(li);
+  });
+
+  bookListDiv.appendChild(bookList);
+}
+
+const form = document.querySelector('#form');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.getElementById('book-title').value;
+  // Capitalize first letter of title value
+  const titleUC = title.charAt(0).toUpperCase() + title.slice(1);
+  const author = document.getElementById('book-author').value;
+  const arrAuthor = author.split(' ');
+  // Capitalize first letter of each word of author value
+  for (let i = 0; i < arrAuthor.length; i += 1) {
+    arrAuthor[i] = arrAuthor[i].charAt(0).toUpperCase() + arrAuthor[i].slice(1);
+  }
+  const authorUC = arrAuthor.join(' ');
+  if ((title && author) !== '') {
+    const book = new Book(titleUC, authorUC);
+    const library = new Library(book);
+    library.addBook();
+    form.reset();
+    displayBooks();
+  }
+});
+
+// List - Link
+const bookList = document.querySelector('.book-list-container');
+const listBtn = document.querySelector('.navList');
+const formContainer = document.querySelector('.form');
+// Add New - Link
+const addNewBtn = document.querySelector('.navNew');
+// Contact - Link
+const contactBtn = document.querySelector('.navContact');
+const contactInfo = document.querySelector('.contact');
+
+function showList() {
+  bookList.style.display = 'block';
+  formContainer.style.display = 'none';
+  contactInfo.style.display = 'none';
+  displayBooks();
+}
```

Comment:
kindly use arrow function instead as it is a better practice

Reply:
Thanks for your advice. 

It's Done